### ISSUE TITLE
Save File Failed (said Read-only by Typora)

### ISSUE THREAD

**Issue by whoami90506** [typora/typora-issues#2316]
I made a new markdown and saved successfully by Typora on March 8th.
Today, I wanted to edit it but Typora failed to save the file.
![save failed](https://user-images.githubusercontent.com/31800500/54805022-99cfd380-4cb0-11e9-990e-14bba02a35a8.JPG)

I've tried to open Typora as administrator, yet I still failed to save files.
Moreover, Sublime 3 can edit and save the same file successfully without administrator privilege.

Typora version: 0.9.68
OS : WIndows10 version1083
log file 
![temp](https://user-images.githubusercontent.com/31800500/54805020-99cfd380-4cb0-11e9-9fdc-0ea5a5865521.JPG)
the file made by Typora on March 8th
![detail](https://user-images.githubusercontent.com/31800500/54805493-23cc6c00-4cb2-11e9-97d5-356cefb73d4e.JPG)

Can anyone help me plz?

**Comment by abnerlee** [comment]
Possible to send us the sample file and it path?

Also, please make sure its parent folders are editable, and the file is not occupied by other apps, or is there any anti-virus software to prevent apps modify the file.

Also, from the icon, the file is synced someway, which app did you use for file syncing, onedrive?

**Comment by whoami90506** [comment]
In fact, it automatically works few days after I wrote this issue, even through I believe that I did nothing during the interval.
Yet I'd still like to send the detail.
Thanks for your help.

[Two-player zero-sum games.txt](https://github.com/typora/typora-issues/files/3101104/Two-player.zero-sum.games.txt)
(github doesn't allow me to upload .md file, so I change the name to .txt)
Path : 
D:\Dropbox\大學\courses\game theory\Two-player zero-sum games.md
As shown in path, I use Dropbox for file syncing.
